lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -1218,6 +1223,35 @@ export async function mergePr({
   return true;
 }
 
+// https://confluence.atlassian.com/bitbucketserver094/merge-a-pull-request-1489802114.html#Mergeapullrequest-Auto-mergeapullrequest
+export async function tryPrAutomerge(
+  prNumber: number,
+  prVersion: number,
+): Promise<void> {
+  logger.debug(`automergePr(${prNumber})`);
+
+  if (semver.lt(defaults.version, '8.15.0')) {
+    logger.debug(
+      { prNumber },
+      'Bitbucket Server-native automerge: not supported on this version of Bitbucket. Use 8.15.0 or newer.',
+    );
+    return;
+  }
+
+  try {
+    const body: PullRequestMerge = { autoMerge: true };
+    await bitbucketServerHttp.postJson(
```

Comment:
don't you need to return the new version of further processing? or drop PR from cache to refetch if needed? 🤔